content/en/project/_index.md

```diff
@@ -13,6 +19,11 @@ We also want to restore and present earlier versions of Interlisp, for the stude
 
 There has been a lot of cleanup and adaptation to make it usable again in the modern world. Among other developments, you can now run Medley Interlisp on many OS and hardware configurations, or at https://online.interlisp.org in the cloud, using a web browser. See our [2021 Annual Report](news/2021medleyannualreport), [Project News](/news/), and [recent release notes](https://github.com/Interlisp/medley/releases) . We've also been working on integration of the Interlisp style development with git and GitHub, Docker and other modern components.
 
+# [How do I try using Interlisp?](using)
```

Comment:
Ia Using a subsection of Project?  If so, this belongs here, if not it should be moved.